lib/wpc_proto/proto_options/wp_global.options

```diff
@@ -1,5 +1,5 @@
 wp.RequestHeader.sink_id max_size:16
-wp.ResponseHeader.gw_id max_size:16
```

Comment:
Internally, to store the gateway id, we're using GATEWAY_ID_MAX_SIZE = 16. So we should increase it also.
There is a check that both size are compatible, but we just check that the target (protobuf) is large enough to receive the internal version. So the check will not fire. It's working, but we still can't use the full size.
I think we can't use directly the size of option file (or defined in generated file) in our code, too bad. We can only test with this kind of static assert. We could check with == instead of >=, but we did it on purpose to avoid errors in case the size was increased on the interface (keep compatibility).
Also, in the main, we have static char m_gateway_id[64]  (that could let customer think the size could be 64). It could be better to use GATEWAY_ID_MAX_SIZE also (it's in common.h).
m_gateway_id is defined both in main.c and common.c. Ok they are static in both case, but it could be better to rename one as main_gateway_id for example, to avoid confusion.

Reply:
Very good, comment, I'll update the constant to reflect the change

Reply:
common.h is internal and not part of the api that can be used from app example. Either we move it to wpc_proto.h or just use same value (what I did in update)